packages/elasticsearch-plugin/e2e/elasticsearch-plugin.e2e-spec.ts

```diff
@@ -913,9 +913,8 @@ describe('Elasticsearch plugin', () => {
                     groupByProduct: true,
                     term: 'gaming',
                 });
-                expect(result.search.items.map(pick(['productId', 'enabled']))).toEqual([
-                    { productId: 'T_3', enabled: false },
-                ]);
+                const t3 = result.search.items.find(i => i.productId === 'T_3');
+                expect(t3?.enabled).toEqual(false);
```

Comment:
Fuzzy matche returns multiple results now, but this test only cares about if T3 is disabled, so we should ignore the other results